infrastructure/docker-registry/README.md

```diff
@@ -1,74 +1,72 @@
 
-  2. Annotations concerning the configuration of the Ingress controller.
-```yaml
-ingress:
-  enabled: true
-  path: /
-  hosts:
-    - <registry.domain.name>
-  annotations:
-    cert-manager.io/cluster-issuer: letsencrypt-production
-  labels: {}
-  tls:
-    - hosts:
-      - <registry.domain.name>
-```
+  1. Annotations concerning the configuration of the Ingress controller. You have to set the type of the ingress (in our file values.yaml the type of the ingress is ingress) and the parameters for the type you have chosen. For example you have to set the name of the hosts of core and notary by setting the ingress.hosts.core and ingress.hosts.notary parameters.
 
-  3. Configuration of the persistent disk. Please dimension this disk appropriately, as VM images may be rather large and hence you may fill up this space in a very short time.
-```yaml
-persistence:
-  accessMode: 'ReadWriteOnce'
-  enabled: true
-  size: 500Gi
-  storageClass: 'rook-ceph-block'
-```
+  2. The external URL to expose the Harbor Registry. To set the external URL you have to set the externalURL parameter
+
+  3. The deployment pods of Portal, Core and Registry to have 3 replicas of each one and to set the anti-affinity of each pod, to have pod of the same services in different nodes. We have also set the resources consumptions after have done a stress test. To do this you have to set respectively the affinity and the resources parameter
+
+  4. For the installation of Postgres service, please see the link on the pre-requisites section.
+
+  5. For the installation of Redis service, please see the link on the pre-requisites section. In our architecture we have a [Redis-Sentinel](https://redis.io/topics/sentinel) service, instead of [Redis Cluster](https://redis.io/topics/cluster-tutorial) by setting the following parameter in the file values of redis_service_values.yaml file. In the file you have to set an IP for the ClusterIP of the sentinel where after harbor is attached in the cluster and a name for your master. If you want have the replicas of redis running in different nodes you have to set the replica.podAntiAffinityPreset parameter to soft or hard, depending what you need. Please see this [link](https://docs.bitnami.com/tutorials/assign-pod-nodes-helm-affinity-rules/) to see the differences.
+  ```yaml 
+  sentinel.enabled=true
+  ```
+  6. For the external redis service you have to modify the redis.type parameter to external and have to set the same IP and the same master name of the previous redis service respectively in redis.external.addr parameter and in redis.external.sentinelMasterSet parameter.
 
 ## Installing the Chart
+Before install the chart, the repository must be add to helm with the following command:
+```bash
+helm repo add harbor https://helm.goharbor.io
+```
 To install the chart, use the following command:
```

Comment:
```suggestion
To install the chart with the release name `harbor` and apply the configuration specified by the `harbor-values.yaml` file, it is possible to proceed as follows:
```

Would directly add the correct command below (the one you used to deploy the manifest), to simplify future upgrades.